README.md

```diff
@@ -2,36 +2,33 @@
 
-![GitHub release](https://img.shields.io/github/release/tumblr/k8s-sidecar-injector.svg) ![Travis (.org)](https://img.shields.io/travis/tumblr/k8s-sidecar-injector.svg) ![Docker Automated build](https://img.shields.io/docker/automated/tumblr/k8s-sidecar-injector.svg) ![Docker Build Status](https://img.shields.io/docker/build/tumblr/k8s-sidecar-injector.svg) ![MicroBadger Size](https://img.shields.io/microbadger/image-size/tumblr/k8s-sidecar-injector.svg) ![Docker Pulls](https://img.shields.io/docker/pulls/tumblr/k8s-sidecar-injector.svg) ![Docker Stars](https://img.shields.io/docker/stars/tumblr/k8s-sidecar-injector.svg) [![Godoc](https://godoc.org/github.com/tumblr/k8s-sidecar-injector?status.svg)](http://godoc.org/github.com/tumblr/k8s-sidecar-injector)
+## What is this?
 
+At Riege, we run some containers that have complicated sidecar setups. A kubernetes pod may run 5+ other containers, with some associated volumes and environment variables. It became clear quickly that keeping these sidecars in line would become an operational hassle; making sure every service uses the correct version of each dependency, updating global environment variable sets as configurations in our DCs change, etc.
 
-# What is this?
-
-At Tumblr, we run some containers that have complicated sidecar setups. A kubernetes pod may run 5+ other containers, with some associated volumes and environment variables. It became clear quickly that keeping these sidecars in line would become an operational hassle; making sure every service uses the correct version of each dependency, updating global environment variable sets as configurations in our DCs change, etc. 
-
-To help solve this, we wrote the `k8s-sidecar-injector`. It is a small service that runs in each Kubernetes cluster, and listens to the Kubernetes API via webhooks. For each pod creation, the injector gets a (mutating admission) webhook, asking whether or not to allow the pod launch, and if allowed, what changes we would like to make to it. For pods that have special annotations on them (i.e. `injector.tumblr.com/request=logger:v1`), we rewrite the pod configuration to include the containers, volumes, volume mounts, host aliases, init-containers and environment variables defined in the sidecar `logger:v1`'s configuration.
+To help solve this, we wrote the `k8s-sidecar-injector`. It is a small service that runs in each Kubernetes cluster, and listens to the Kubernetes API via webhooks. For each pod creation, the injector gets a (mutating admission) webhook, asking whether or not to allow the pod launch, and if allowed, what changes we would like to make to it. For pods that have special annotations on them (i.e. `injector.riege.com/request=logger:v1`), we rewrite the pod configuration to include the containers, volumes, volume mounts, host aliases, init-containers and environment variables defined in the sidecar `logger:v1`'s configuration.
```

Comment:
We didn't write this initially, maybe we should give credit to Tumblr for most of the implementation here?

Reply:
Right, I have added another documentation update.